src/Controllers/ShareDraftController.php

```diff
@@ -50,6 +51,9 @@ class ShareDraftController extends Controller
      */
     public function preview(HTTPRequest $request)
     {
+        // Ensure this URL doesn't get picked up by HTTP caches
+        HTTPCacheControlMiddleware::singleton()->disableCache();
```

Comment:
Why not private cache?

Reply:
Being conservative? Sure, you could treat repeat requests from the same browser as cacheable within this browser, in case something else in the request processing set a max age. But that's pretty low value. The important bit is to ensure there's no caching, and the default way of achieving this is through `disableCache()`. If you feel strongly about this, feel free to change it to `privateCache()` (for both 1.x and 2.x PRs)